integrity-backend/input-proofmode-signal-examples/3e11cc57daf3bad8375935cad4878123acc8d769551ff90f1b1bb0dc597-meta-recorder.json

```diff
@@ -0,0 +1,50 @@
+{
+  "recorderMetadata": [
+    {
+      "service": "api",
+      "info": [
+        {
+          "type": "external",
+          "values":
+          {
+            "name": "proofmode-1.0.0"
+          }
+        },
+        {
+          "type": "docker",
+          "values":
+          {
+            "name": "signal_bot",
+            "image": "sha256:b560c010a66d20bdc79724b9f233c49dfa1211db42740ddb0763f92112017ae9"
+          }
+        },
+        {
+          "type": "git",
+          "values":
+          {
+            "repository": "https://github.com/starlinglab/signal-bot.git",
+            "branch": "main",
+            "commit": "c45b3e6650ad880048cfd7baf7d712582f316768",
+            "clean": true
+          }
+        }
```

Comment:
Waiting on this.

Reply:
```
        {
            "service": "signal_bot_main",
            "info": [
                {
                    "type": "docker",
                    "values": {
                        "name": "signal_bot",
                        "image": "sha256:3fc85bbca2cf9f15ea9293864eaad72121fe5e27e8e126b622135b5b679cdbe3"
                    }
                },
                {
                    "type": "docker",
                    "values": {
                        "name": "signald",
                        "image": "sha256:0ddb4b3d4f0c62d03709f737735eee5c88a3d7c8a40fcc7a8011cdbfba8db397"
                    }
                },
                {
                    "type": "git",
                    "values": {
                        "repository": "https://github.com/starlinglab/signal_bot.git",
                        "branch": "main",
                        "commit": "3bbfd9db6f16ae11b4f09974a33de0a7349d5fb9",
                        "clean": false
                    }
                }
            ]
        },
        {
            "service": "integrity-preprocessor",
            "info": [
                {
                    "type": "git",
                    "values": {
                        "repository": "https://github.com/starlinglab/integrity-preprocessor.git",
                        "branch": "dev",
                        "commit": "9a3afcd8d698ebb1cde1484dbd5703209f9baa19",
                        "clean": false
                    }
                }
            ]
        }
```

Reply:
Might need to add another manual git for signald since its pulled

Reply:
```
        {
            "service": "signal_bot_main_signald",
            "info": [
                {
                    "type": "git",
                    "values": {
                        "repository": "https://gitlab.com/signald/signald.git",
                        "branch": "main",
                        "commit": "c5893713f82eaf735db59dc606c738f866b27a3a",
                        "clean": false
                    }
                }
            ]
        },

```